SimpleKeychain/A0SimpleKeychain.m

```diff
@@ -356,12 +356,12 @@ - (NSDictionary *)queryFetchOneByKey:(NSString *)key message:(NSString *)message
                                       (__bridge id)kSecAttrAccount: key,
                                       }];
 #if TARGET_OS_IPHONE
-    if (self.useAccessControl && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
-        if (message) {
+    if (self.useAccessControl) {
+        if (message && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
             query[(__bridge id)kSecUseOperationPrompt] = message;
+        } else {
+            NSLog(@"Error trying to access to non available kSecUseOperationPrompt in iOS7");
```

Comment:
The problem is that when self.useAccessControl == NO and the iOS version is greater than 7.1, the log message always appears when this method is called.

You should fix this by breaking the first if statement into 2 nested if statements:

    if (self.useAccessControl) {
        if (floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
            if (message) {
                query[(__bridge id)kSecUseOperationPrompt] = message;
            }
        } else {
            NSLog(@"Error trying to access to non available kSecUseOperationPrompt in iOS7");
        }
    }